allvers/2005-05-08-GPL-2.0-from-gnu.org.txt.notes

```diff
@@ -0,0 +1,5 @@
+Date: 2005-05-08
```

Comment:
Although very likely it was this date I could only find a range between May 1 - 8.

Reply:
Updated in b0b4d776c3f9a4d7e98e16830c7c64a623af547e